tests/integration/cli/projects/test/tests/test_fixtures.py

```diff
@@ -12,6 +12,10 @@ def test_chain(chain):
     assert isinstance(chain, ChainManager)
 
 
+def test_contract(Contract):
+    assert isinstance(Contract, ContractInstance)
```

Comment:
no this is callable, this test will fail (if you run it)...
you need to get an address from somewhere else and then `Contract(address)` will be the instance